libs/extensions/angular/image-banner/src/image-banner.component.stories.ts

```diff
@@ -41,6 +41,9 @@ const meta: Meta<ImageBannerComponent> = {
     dismissClick: {
       control: false,
     },
+    onImageError: {
```

Comment:
```suggestion
    imageError: {
```

Reply:
The onImageError method is excluded above, so this config has no effect. 
Instead it should set the control of the `imageError` output emitter to false, so the textarea in the below image is removed.

<img width="1029" alt="image" src="https://github.com/user-attachments/assets/211a5b9b-7481-4b90-91cb-a52fb6cffe0d" />